packages/lesswrong/lib/collections/chapters/collection.ts

```diff
@@ -35,6 +35,10 @@ export const Chapters: ChaptersCollection = createCollection({
 
 export const makeEditableOptions = {
   order: 30,
+  getLocalStorageId: (chapter, name) => {
+    if (chapter._id) { return {id: `${chapter._id}_${name}`, verify: true} }
+    return {id: `sequence: ${chapter.sequenceId}_${name}`, verify: false}
```

Comment:
What's the reasoning behind `verify: true` when the chapter has an ID and `verify: false` when it doesn't?

Reply:
The idea is that when it has an id, then it has existing content, and we somehow need to give you the choice of resetting the editor to whatever it's current content is. When it doesn't have an id it doesn't have existing content, and as such, there is nothing we are overwriting when we restore it.